language/move-prover/bytecode-to-boogie/tests/translator_tests.rs

```diff
@@ -38,11 +39,9 @@ fn compile_files(file_names: Vec<String>) -> (Vec<VerifiedModule>, SourceMap<Loc
     (verified_modules, source_maps)
 }
 
-#[test]
-fn test3() {
+fn generate_boogie(file_name: String) -> String {
```

Comment:
You could also put this into a file `tests/boogie/mods.rs` and then use here `use boogie::*;`. Cargo will recognize this and compile this as a helper module into each integration test. With this you can split the tests into multiple files. Doesn't look like this is needed now put perhaps in the future.

Reply:
Sounds good.